# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qjackctl/compile-fix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Nur qjackctl-0.2.18.orig/src/qjackctlConnectionsForm.ui qjackctl-0.2.18/src/qjackctlConnectionsForm.ui
--- qjackctl-0.2.18.orig/src/qjackctlConnectionsForm.ui	2005-01-27 18:11:46.000000000 +0200
+++ qjackctl-0.2.18/src/qjackctlConnectionsForm.ui	2005-11-01 21:48:35.000000000 +0200
@@ -447,7 +447,5 @@
 <pixmapinproject/>
 <layoutdefaults spacing="4" margin="4"/>
 <includehints>
-    <includehint>qjackctlconnectview.h</includehint>
-    <includehint>qjackctlconnectview.h</includehint>
 </includehints>
 </UI>
diff -Nur qjackctl-0.2.18.orig/src/qjackctlPatchbayForm.ui qjackctl-0.2.18/src/qjackctlPatchbayForm.ui
--- qjackctl-0.2.18.orig/src/qjackctlPatchbayForm.ui	2005-05-04 14:30:52.000000000 +0300
+++ qjackctl-0.2.18/src/qjackctlPatchbayForm.ui	2005-11-01 21:49:28.000000000 +0200
@@ -942,6 +942,5 @@
 <pixmapinproject/>
 <layoutdefaults spacing="4" margin="4"/>
 <includehints>
-    <includehint>qjackctlpatchbayview.h</includehint>
 </includehints>
 </UI>
